app/javascript/react/screens/App/Plan/Plan.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Breadcrumb, Spinner } from 'patternfly-react';
+import Toolbar from '../../../config/Toolbar';
+import PlanRequestDetailList from './components/PlanRequestDetailList';
+
+class Plan extends React.Component {
+  // constructor(props) {
+  //   super(props);
+
+  // bindMethods(this, ['stopPolling', 'startPolling']);
+  // }
+
+  componentDidMount() {
+    const { fetchPlanRequestsUrl, fetchPlanRequestsAction } = this.props;
+
+    fetchPlanRequestsAction(fetchPlanRequestsUrl);
+
+    // this.startPolling();
+  }
+
+  // componentWillUnmount() {
+  //   this.stopPolling();
+  // }
+
+  // startPolling() {
+  //   const {
+  //     fetchPlanRequestsAction,
+  //     fetchPlanRequestsUrl
+  //   } = this.props;
+  //   this.pollingInterval = setInterval(() => {
+  //     fetchPlanRequestsAction(fetchPlanRequestsUrl);
+  //   }, 15000);
+  // }
+
+  // stopPolling() {
+  //   if (this.pollingInterval) {
+  //     clearInterval(this.pollingInterval);
+  //     this.pollingInterval = null;
+  //   }
+  // }
+
+  render() {
+    const {
+      planRequestDetails,
+      isRejectedPlanRequests,
+      isFetchingPlanRequests
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <Toolbar>
+          <Breadcrumb.Item href="/dashboard/maintab?tab=compute">
+            Compute
+          </Breadcrumb.Item>
+          <Breadcrumb.Item href="/migration">Migration</Breadcrumb.Item>
+          {!isFetchingPlanRequests &&
+            !isRejectedPlanRequests &&
+            planRequestDetails.name && (
+              <Breadcrumb.Item active>
+                {planRequestDetails.name}
+              </Breadcrumb.Item>
+            )}
+        </Toolbar>
+        <div style={{ overflow: 'auto', paddingBottom: 1, height: '100%' }}>
+          <Spinner loading={isFetchingPlanRequests}>
+            {!isFetchingPlanRequests &&
+              !isRejectedPlanRequests &&
+              planRequestDetails.tasks &&
```

Comment:
Consider adding tasks as an empty array to the `defaultProps` object.  It provides more default props context and simplifies the logic here, no need to check if it is defined, only that the length is > 0